app/templates/views/public/application_review.html

```diff
@@ -3,23 +3,23 @@
 <h2 style="text-align:center;">Gallery</h2>
```

Comment:
The gallery object is part of the form table, so we can use that to get the current gallery title.